Traceback (most recent call last):
  File "/home/vagrant/.pyenv/versions/3.8.10/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/vagrant/.pyenv/versions/3.8.10/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/vagrant/ashiq_hypatia/satgenpy/satgen/post_analysis/main_print_routes_and_rtt.py", line 53, in <module>
    main()
  File "/home/vagrant/ashiq_hypatia/satgenpy/satgen/post_analysis/main_print_routes_and_rtt.py", line 41, in main
    print_routes_and_rtt(
  File "/home/vagrant/ashiq_hypatia/satgenpy/satgen/post_analysis/print_routes_and_rtt.py", line 73, in print_routes_and_rtt
    with open(satellite_network_dynamic_state_dir + "/fstate_" + str(t) + ".txt", "r") as f_in:
FileNotFoundError: [Errno 2] No such file or directory: '../ns3-sat-sim/simulator/my-test/large-starlink/satellite_network_state/gen_data/starlink_550_isls_gs_top_100_algorithm_free_gs_one_sat_many_only_over_isls/dynamic_state_100ms_for_500s/fstate_0.txt'
Data dir: ../ns3-sat-sim/simulator/my-test/satgen_analysis/data/
Used data dir to form base output dir: ../ns3-sat-sim/simulator/my-test/satgen_analysis/data//starlink_550_isls_gs_top_100_algorithm_free_gs_one_sat_many_only_over_isls/100ms_for_500s/manual
